exchanges/stream/websocket.go

```diff
@@ -984,7 +983,7 @@ func (w *Websocket) CanUseAuthenticatedEndpoints() bool {
 }
 
 // isDisconnectionError Determines if the error sent over chan ReadMessageErrors is a disconnection error
```

Comment:
Fix lint 🦀

Reply:
Sorry about that.
Didn't (and still doesn't) report locally.
I'll workout why later but I've fixed it anyway.